tomcat7/image.yaml

```diff
@@ -57,8 +57,6 @@ modules:
 packages:
       repositories:
           - jboss-os
-      install:
-          - apr-util
```

Comment:
I think it's safe to remove the whole packages element.  Same goes for tomcat8/image.yaml below.